org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/BuildWorkspaceStatus.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat Inc. and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Microsoft Corporation - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.jdt.ls.core.internal;
+
+/**
+ * @author xuzho
+ *
+ */
+public enum BuildWorkspaceStatus {
```

Comment:
previously i thought about canceled, but seems client can not cancel build now, so i leave it out. leave it here maybe later we need it?